### PR TITLE
Fix test_lib on Python 3.8.10 due to importlib relative path changes

### PR DESCRIPTION
**Pull request by benhoyt**
Now that GitHub Actions is on Python 3.8.10, test_lib.test_cwd fails.
This is due to Python 3.8.10 changing (fixing?) importlib to resolves
relative paths when creating module spec objects from file locations.

This change should work on both 3.8.9 and 3.8.10.

See:
* https://docs.python.org/release/3.8.10/whatsnew/changelog.html#core-and-builtins
* https://bugs.python.org/issue43105

### CODE REVIEW

**Review by hpidcock — APPROVED** [review]

**Comment by benhoyt** [comment]
Thanks @hpidcock.

I'm going to merge this before getting 2 approvals as CI is currently broken on master.

**Pull request merged**
